2022/Q2/artifacts/pos-model/PoS-model.md

```diff
@@ -209,21 +198,26 @@ func bond(validator_address, delegator_address, amount, offset_length)
 ```
 
 ```go
+/* COMMENT
```

Comment:
Because of line 193 above, where we apply `total_deltas_at` to the voting power, consider renaming `total_deltas_at` to something more general

Reply:
yes, you're right. Solved